app/assets/stylesheets/sequence_builder.css

```diff
@@ -209,8 +202,40 @@ span.prefix{
   font-weight: bold;
 }
 
-/* SITE-WIDE ISH */
+/* SITE-WIDE BACKGROUND */
 
 body.ng-scope {
 background: #EEE;
+}
+
+/* SITE WIDE STUFF */
+
+.green-button{
+  font-size: 0.9rem;
+  float: right;
+  background-color: #6AA84F;
+  font-weight: bold;
+  text-transform: uppercase;
+  color: white;
+  padding-top: 7px;
+  padding-bottom: 7px;
+  padding-left: 15px;
+  padding-right: 15px;
+  border: none;
+  border-radius: 5px;
+}
+.red-button{
+  background-color: #E06666;
+}
+
+fieldset legend {
+background: transparent;
+}
+
+fieldset {
+border: none;
+}
+
+.row {
+  max-width: 80rem;
 }
```

Comment:
If you use sublime, consider adding [this config](https://gist.github.com/RickCarlino/8c3768988e8915a033b5) to your user settings, as missing carriage returns can cause issues in some cases. It will also strip trailing white space and convert tabs to spaces.